model/otel/sampling.yaml

```diff
@@ -0,0 +1,49 @@
+groups:
+  - id: sampling.common
+    type: attribute_group
+    brief: 'Semantic conventions describing sampling in OpenTelemetry.'
+    prefix: sampling
+    attributes:
+      - id: priority
+        type: int
+        brief: Enables prioritizing and de-prioritizing collection of telemetry.
+        note: >
+          This attribute is useful to express the user's desire to
+          collect telemetry, despite the sampling configuration in
+          place.  Non-zero values MAY prioritize the item for
+          collection independent of probability sampling.  Zero values
+          have the reverse effect and MAY cause telemetry not to be
+          collected.  This convention derives from a conventional tag
+          with the same name in the OpenTracing specification.
+        examples: [1, 0]
+        tag: otel-span-attributes
```

Comment:
is it necessary ?

Reply:
Removed. Now there is one table of attributes, besides which @kentquirk thinks the attributes should be useful for spans, too.